pkg/controller/vpcpeering/peering.go

```diff
@@ -464,3 +505,16 @@ func (e *external) deleteVPCPeeringConnection(ctx context.Context, cr *svcapityp
 
 	return nil
 }
+
+func (e *external) isInternalVpcPeering(cr *svcapitypes.VPCPeeringConnection) (bool, error) {
+	acountID, err := e.iamClient.GetAccountID()
+	if err != nil {
+		return false, err
+	}
+	// if requester peering connection acountID same with accepter vpc peering connection, auto-accept it
```

Comment:
accepter -> acceptor

Reply:
accepter already define in struct, i think we should be keep consistent
```go
// VPCPeeringConnectionObservation defines the observed state of VPCPeeringConnection
type VPCPeeringConnectionObservation struct {
	// Information about the accepter VPC. CIDR block information is only returned
	// when describing an active VPC peering connection.
	AccepterVPCInfo *VPCPeeringConnectionVPCInfo `json:"accepterVPCInfo,omitempty"`
	// The time that an unaccepted VPC peering connection will expire.
	ExpirationTime *metav1.Time `json:"expirationTime,omitempty"`
	// Information about the requester VPC. CIDR block information is only returned
	// when describing an active VPC peering connection.
	RequesterVPCInfo *VPCPeeringConnectionVPCInfo `json:"requesterVPCInfo,omitempty"`
	// The status of the VPC peering connection.
	Status *VPCPeeringConnectionStateReason `json:"status,omitempty"`
	// Any tags assigned to the resource.
	Tags []*Tag `json:"tags,omitempty"`
	// The ID of the VPC peering connection.
	VPCPeeringConnectionID *string `json:"vpcPeeringConnectionID,omitempty"`

	Phase *string `json:"phase,omitempty"`
}
```